docs/HACKING.md

```diff
@@ -24,3 +24,11 @@ pip install --user -e .
 ```console
 python3.7 setup.py test
 ```
+# Check the Report for Unit Test Coverage
+```
+cd dffml
+coverage run -m unittest discover
+coverage report
+coverage html
+```
+These command will generate a folder ```htmlcov```, you can check the report by opening the ```index.html``` in a web browser. 
```

Comment:
nit: Please add a newline between the codeblock and this sentence. Also please keep this to 80 columns.

## Vim instructions for 80 columns

> Set `textwidth` to 80 (`:set textwidth=80` **(pdxjohnny) do this first**), move to the start of the file (can be
> done with `Ctrl-Home` or `gg`), and type `gqG`.

> `gqG` formats the text starting from the current position and to the end of the
> file. It will automatically join consecutive lines when possible. You can place a
> blank line between two lines if you don't want those two to be joined together.

> `gqq` formats current line. **(pdxjohnny) Then use this one**

Source: https://stackoverflow.com/a/3033455